inc/composer/class-command.php

```diff
@@ -320,18 +335,135 @@ protected function logs( InputInterface $input, OutputInterface $output ) {
 	protected function shell( InputInterface $input, OutputInterface $output ) {
 		$columns = exec( 'tput cols' );
 		$lines = exec( 'tput lines' );
+		$command_prefix = $this->get_base_command_prefix();
 		passthru( sprintf(
-			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash',
-			'vendor/altis/local-server/docker',
-			escapeshellarg( getcwd() ),
-			$this->get_project_subdomain(),
+			"$command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d php /bin/bash",
 			$columns,
 			$lines
 		), $return_val );
 
 		return $return_val;
 	}
 
+	protected function db( InputInterface $input, OutputInterface $output ) {
+		$db = $input->getArgument( 'options' )[0] ?? '';
+		$columns = exec( 'tput cols' );
+		$lines = exec( 'tput lines' );
+
+		$base_command_prefix = $this->get_base_command_prefix();
+
+		$base_command = sprintf(
+			"$base_command_prefix docker-compose exec -e COLUMNS=%d -e LINES=%d db",
+			$columns,
+			$lines
+		);
+
+		$return_val = 0;
+
+		switch ( $db ) {
+			case 'info':
+				$connection_data = $this->get_db_connection_data();
+
+				$db_info = <<<EOT
+<info>Root password</info>:  ${connection_data['MYSQL_ROOT_PASSWORD']}
+
+<info>Database</info>:       ${connection_data['MYSQL_DATABASE']}
+<info>User</info>:           ${connection_data['MYSQL_USER']}
+<info>Password</info>:       ${connection_data['MYSQL_PASSWORD']}
+
+<comment>Version</comment>:        ${connection_data['MYSQL_VERSION']}
+<comment>Host</comment>:          ${connection_data['HOST']}
+<comment>Port</comment>:          ${connection_data['PORT']}
+
+EOT;
+				$output->write( $db_info );
+				break;
+			case 'spf':
+				$connection_data = $this->get_db_connection_data();
+				$rpf_file_contents = file_get_contents( getcwd() . '/vendor/altis/local-server/templates/sequel.xml' );
+				foreach ( $connection_data as $field_name => $field_value ) {
+					$rpf_file_contents = preg_replace( "/(<%=\s)($field_name)(\s%>)/i", $field_value, $rpf_file_contents );
+				}
+				$output_file_path = getcwd() . '/sequel.spf';
```

Comment:
This will get dumped into the root directory so would need to be git ignored, we can avoid this by writing to a temp file instead - the `/tmp` directory on linux based systems is cleared on every reboot so it's a good place to drop things that can be cleaned up automatically. Let's put this in `sprintf( '/tmp/%s.spf', $this->get_project_subdomain() )`